### PR TITLE
Create Brianna's final milestones

### PR DESCRIPTION
**Pull request by brianna-stedman**
final milestone update and snapshot of code

### CODE REVIEW

File: _posts/brianna-stedman/2020-12-02-brianna-stedman-milestones-3.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+author: brianna-stedman
+title: Brianna's final project milestones, last revision
```
Comment (MishaCienfuegos) [inline _posts/brianna-stedman/2020-12-02-brianna-stedman-milestones-3.md]: Would you mind enclosing your title in `"`s? Some characters are invalid outside of a string, and I think `'` is one of them.

**Review by MishaCienfuegos — CHANGES_REQUESTED** [review]
This PR is looking great! Just one small request in the header and it should be good to go.

File: _posts/brianna-stedman/2020-12-02-brianna-stedman-milestones-3.md (hunk 1 above)
Comment (brianna-stedman) [reply _posts/brianna-stedman/2020-12-02-brianna-stedman-milestones-3.md]: shoot, good catch, I'll update it!

Comment (brianna-stedman) [reply _posts/brianna-stedman/2020-12-02-brianna-stedman-milestones-3.md]: updated, I think it's ready to merge :-) thank you so much!

**Review by pccisme — APPROVED** [review]
Looks good!

**Review by MishaCienfuegos — APPROVED** [review]
Looks good, thanks for this change!

**Pull request merged**